<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmDialogWidget</class>
 <widget class="QWidget" name="ConfirmDialogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>6</number>
   </property>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="buttons_layout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="continue_button">
       <property name="text">
        <string>Continue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="interleave_hlayout">
     <item>
      <widget class="QCheckBox" name="interleave_cbx">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Interleave:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="interleave_images_num_ledit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="summary_gbox">
     <property name="title">
      <string>Summary</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="summary_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTreeWidget" name="summary_treewidget">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="take_snapshots_hlayout">
     <item>
      <widget class="QLabel" name="take_snapshots_label">
       <property name="text">
        <string>Number of crystal snapshots:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="take_snapshots_combo">
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="skip_existing_images_cbx">
     <property name="text">
      <string>Skip already collected images</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="force_dark_cbx">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Force dark current</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="file_gbox">
     <property name="title">
      <string>Existing files that will be overwritten:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="file_treewidget">
        <attribute name="headerVisible">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Sample</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Directory</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>File name</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>interleave_cbx</sender>
   <signal>toggled(bool)</signal>
   <receiver>interleave_images_num_ledit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
